tools/p2p_ansible_test/files/detd_listener.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import subprocess
+from detd import *
+
+def clean_up(interface_name, package_name):
+    """
+    Clean up the system by deleting network interfaces and traffic control settings,
```

Comment:
It seems currently the script is not deleting interfaces or settings, just reinstalling the package.